funnel/views/proposal.py

```diff
@@ -299,10 +299,22 @@ def proposal_view(profile, space, proposal):
     else:
         statusform = None
 
+    movable_spaces = []
+    proposal_move_form = None
+    if 'move-proposal' in space.permissions(g.user, g.permissions):
+        team_ids = [t.id for t in g.user.teams]
+        movable_spaces = ProposalSpace.query.join(ProposalSpace.profile).filter(
+            (ProposalSpace.admin_team_id.in_(team_ids)) |
+            (Profile.admin_team_id.in_(team_ids))
+            ).order_by(ProposalSpace.date.desc())
+
+        proposal_move_form = ProposalMoveForm()
+        proposal_move_form.target.choices = [(space.id, space.title) for space in movable_spaces]
```

Comment:
Use `set_queries` inside the form to set choices. See https://github.com/hasgeek/boxoffice/blob/master/boxoffice/forms/discount.py#L50 for an example.

Reply:
@shreyas-satish that's for `QuerySelectMultipleField`, this is a basic `SelectField`.